docs/methoddocs/filleeventqueries.md

```diff
@@ -1 +1,300 @@
 # File Event Queries
+
+```eval_rst
+.. autoclass:: py42.clients.file_event.FileEventClient
+    :members:
+    :show-inheritance:
+```
+
+```eval_rst
+.. autoclass:: py42.sdk.queries.fileevents.file_event_query.FileEventQuery
+    :members:
+    :show-inheritance:
+```
+
+# Filter Classes
+
+Each of the following classes can be used to filter Forensic Search results based on the class's corresponding event
+attribute. Call the appropriate classmethod on your desired filter class with the `value` you want to match, and it
+will return a FilterGroup object that can be passed to `FileEventQuery`'s `all()` or `any()` methods to combine complex
```

Comment:
Little thing, but GitHub does not like the wrapping here, "complex" could be on a new line to remedy that.